JS/Anna_G/TaskOfBookStore/src/index.js

```diff
@@ -0,0 +1,65 @@
+import _ from 'lodash';
+import data from '../resources/books.json'
+
+const formBlock = document.querySelector('#form-block');
+const book = document.querySelector('#bookdiv');
+const globalDiv = document.getElementsByClassName('a');
+
+function createElements(type, newClassName) {
+    const variableName = document.createElement(type);
+    variableName.className = newClassName;
+    return variableName;
+}
+
+function initializedBookElements(index, bookElements){
+    bookElements.titleH.textContent = data[index].title;
+    bookElements.authorH.textContent = data[index].author;
+    bookElements.elem.src = data[index].imageLink;
+    bookElements.button.innerHTML = "Delete";
+}
+
+function appendBook(book, index) {
+    const bookElements = createBookElements();
+    const column = createElements('div', 'column');
+    initializedBookElements(index,bookElements);
+    let key;
+    for (key in bookElements) {
+        column.appendChild(bookElements[key])
+        console.log(key);
```

Comment:
Never keep console.log in regular code, it make JS work slower a lot. Comment delete before submitting for review